source/ddbus/thin.d

```diff
@@ -264,6 +267,66 @@ struct DBusAny {
       ~ ", " ~ valueStr ~ ")";
   }
 
+  /++
+    Returns the value stored in the DBusAny object by specifying the type.
```

Comment:
you should change this to `Returns: ...` because ddoc supports that as special syntax